.github/workflows/oppiabot.yml

```diff
@@ -15,5 +14,3 @@ jobs:
         uses: oppia/oppiabot@1.0.0
```

Comment:
In the line above -- could you please make the "name" much clearer, i.e. have it be a clear description of what action is being performed? (See the other workflow files for examples.) When reading the logs I had no idea what was going on and this hindered debugging.

Reply:
Done.

Reply:
Please make it even clearer. What are you checking about issue labels?

Reply:
Done.